app/javascript/components/util/InfoTooltip.js

```diff
@@ -374,6 +374,10 @@ class InfoTooltip extends React.Component {
         <span data-for={this.customID} data-tip="" className="ml-1">
           <i className="fas fa-question-circle px-0"></i>
         </span>
+        <div className="sr-only">
+          {this.props.tooltipTextKey && <span>Tooltip: {TOOLTIP_TEXT[this.props.tooltipTextKey]}</span>}
```

Comment:
Read the tooltip text (as the tooltip is COMPLETELY left out of the accessibility tree right now)